webapp/tableWithOdp/tableWithOdp.js

```diff
@@ -69,7 +68,7 @@ export class TableWithOdp extends React.Component {
   }
 
   render () {
-    const { copyValues = true, disabled = false, sectionAnchor, i18n } = this.props
+    const { copyValues = true, disabled = false, sectionAnchor, i18n, userInfo } = this.props
```

Comment:
can we use defaultProps for default values?

Reply:
refactor will come later